action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Install dfx'
+description: 'Install the dfx utility at a particular version'
+inputs:
+  dfx-version:
+    description: >
+      The dfx version to install. This value will be used to populate
+      the DFX_VERSION variable that will be handled by the installation
+      script "install.sh".
+    default: 'latest'
```

Comment:
Right now `latest` refers to the last _released_ dfx version, not the last _promoted_ dfx version.  I understand that we plan to change these semantics in the future, but for the time being, this will have to be a fixed version.